### PR TITLE
[CARBONDATA-3822] Fixed load time taken and added format back

### PR DESCRIPTION
**Pull request by kunal642**
### Why is this PR needed?
1.  Load time taken is shown as  PT-1.0S which is wrong(negative time)
2. Show segment is missing format information.

 ### What changes were proposed in this PR?
1. Fix the time and remove PT to show the time as "1.0S" for better viewing.
2. Added format back
   
 ### Does this PR introduce any user interface change?
 - No
 - Yes. (please explain the change and update document)

 ### Is any new testcase added?
 - No
 - Yes

### CODE REVIEW

File: integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/segment/ShowSegmentTestCase.scala [hunk 1]
```diff
@@ -187,6 +187,16 @@ class ShowSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     dropTable(tableName)
   }
 
+  test("test for load time and format name") {
```
Comment (akashrn5) [inline integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/segment/ShowSegmentTestCase.scala:190]: can you add one test case for file format info in show segments in addsegment test cases?

Comment (kunal642) [reply integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/segment/ShowSegmentTestCase.scala:190]: added in AddSegmentTestCase

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1300/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/3018/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1306/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/3024/

**Comment by akashrn5** [comment]
LGTM

**Pull request closed without merging**